docs/visualization/visualization_api/pub_sub_information.md

```diff
@@ -1,22 +1,31 @@
 # Publish/Subscription Information
 
-CARET can visualize the publish/subscribe frequency and period.
-Execute these commands in advance.
+CARET can visualize the publish/subscription frequency and period.
+This document describes sample visualization scripts for them.
+Execute these commands before each section.
 
 ```python
 from caret_analyze.plot import Plot
 from caret_analyze import Application, Architecture, Lttng
 from bokeh.plotting import output_notebook, figure, show
-output_notebook()
+output_notebook() # Note 1
 
 arch = Architecture('yaml', '/path/to/architecture_file')
 lttng = Lttng('/path/to/trace_data')
 app = Application(arch, lttng)
 pub = app.get_publishers('topic_name')
-pub = pub[0]
-# or sub = app.get_subscirber('pub_node', 'sub_node', 'topic_name')
+pub = pub[0] # Note 2
+
+# Please use the following commands in the case visualizing subscription.
+# sub = app.get_subscriptions('topic_name')
+# sub = sub[0]
 ```
 
+Note:
+
+1. The `output_notebook()` is needed only when running the script on Jupyter Notebook.
+2. The function `get_publishers()` and `get_subscriptions()` returns a List of publisher/subscriber having the argument topic. The sample code uses the first value of the List.
```

Comment:
@takeshi-iwanari 
I adapted your comment
https://github.com/tier4/CARET_doc/pull/95#discussion_r999193972
> It will be kind for readers why it returns list and which one (index) should be used.

I added explanation.